db/migrate/20151435234635_add_container_node_max_pods.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerNodeMaxPods < ActiveRecord::Migration
+  def change
+    add_column :container_nodes, :maxpods, :string
```

Comment:
@oschreib in the backend I wouldn't start to introduce the `pods` nomenclature. This should probably be `max_allowed_groups` or `max_container_groups` or `max_allowed_container_groups` (but I fear this one is too long)